### DISCUSSION TITLE
awesome app with a major flaw

### DISCUSSION THREAD

**Discussion by MarTen-10** [File-New-Project/EarTrumpet#1303] [Ideas]
so because I have a costume build pc I have so many audio outputs that I don't use. would be great if I can pick which one to show on the bottom of the taskbar instead of having to expand it all and see so many audio outputs to change my volume. make an option to pick which one to show and be able to hide the rest when it's minimized form. hope you can understand what I mean.

**Comment by Xyncgas** [comment]
Easy

**Reply by Xyncgas** [reply]
I miss having this in windows xp

**Comment by danbrakeley** [comment]
Not sure if this is what you are looking for, but I've noticed that if I open the old Windows Control Panel (and NOT the modern Windows Settings) and then open the Sound panel, I can right click on devices I don't care about and click "Disable", and that prevents them from showing up in apps like EarTrumpet.

![image](https://user-images.githubusercontent.com/6772870/216773745-3c538609-b7cc-4f63-bc0c-5dfeae9cd7c7.png)
